testsuite/gnat2goto/tests/quantified_expressions/test.opt

```diff
@@ -0,0 +1 @@
+ALL XFAIL gnat2goto crashes on quantified expression parsing
```

Comment:
We will have to be very careful with this as it can be very difficult to support quantifiers but it should be possible to at least parse these and generate the appropriate irept.